src/test/java/com/bobocode/hoverla/bring/context/BeanConfigurationClassScannerTest.java

```diff
@@ -0,0 +1,64 @@
+package com.bobocode.hoverla.bring.context;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+class BeanConfigurationClassScannerTest {
+
+    private static final String CONFIG_PACKAGE_NAME = "com.bobocode.hoverla.bring.testsubject.scanner";
+    private static final String NO_CONFIG_PACKAGE_NAME = "com.bobocode.hoverla.bring.testsubject.scanner.empty";
+    private BeanConfigClassValidator validator;
+    private BeanDefinitionMapper mapper;
+    private BeanConfigurationClassScanner scanner;
+
+    @BeforeEach
+    void setUp() {
+        validator = mock(BeanConfigClassValidator.class);
```

Comment:
Since we already have `mockito-junit-jupiter` dependency, would you mind removing this method and simply use `@Mock` annotation on `maper` and `validator`?

Reply:
I don't mind

Reply:
Fixed